### PR TITLE
feat(ImJoy): Support context config

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
xref https://github.com/Kitware/itk-vtk-viewer/pull/355

File: doc/content/docs/imjoy.md [hunk 1]
```diff
@@ -47,12 +47,18 @@ function encodeScijsArray(array){
 
 The `image` key is optional; one can also call `setImage()` later.
 
+Context `config`:
```
Comment (oeway) [inline doc/content/docs/imjoy.md:50]: Maybe list all the available config options?

**Review by oeway — COMMENTED** [review]
This is awesome! I have only one comment, maybe we should document what are available options.

File: doc/content/docs/imjoy.md (hunk 1 above)
Comment (thewtex) [reply doc/content/docs/imjoy.md:50]: @oeway great idea! :bulb: Yes, added in 63d240e. Since we will incrementally add many fields, I created a new page.

Comment (oeway) [reply doc/content/docs/imjoy.md:50]: Great!

**Review by oeway — APPROVED** [review]
Looks good, thank you!

**Pull request merged**